test/maybe.test.js

```diff
@@ -93,3 +93,29 @@ describe('Maybe', function() {
     });
 
 });
+
+describe('Maybe usage', function() {
+
+  it('should allow the user to check if the instance is a Nothing', function() {
```

Comment:
Seems to make sense. 

I think there's a law for `getOrElse` also:

```javascript
jsv.assert(jsv.forall(m, 'nat', function(m1, n) {
  if (m1.getOrElse(n) !== n) {
    return m1.isJust();
  } else {
    return m1.isNothing() || m1.equals(Maybe.Just(n));
});
```

I wonder if it can be simplified though.